core/services/pipeline/orm_test.go

```diff
@@ -715,40 +823,52 @@ func Test_Prune(t *testing.T) {
 
 	ps1 := cltest.MustInsertPipelineSpec(t, db)
 
+	// We need a job_pipeline_specs entry to test the pruning mechanism
+	_, err := db.Exec(`SET CONSTRAINTS fk_job_pipeline_spec_job DEFERRED`)
+	require.NoError(t, err)
+	_, err = db.Exec(`INSERT INTO job_pipeline_specs (job_id,pipeline_spec_id, is_primary) VALUES ($1, $2, false)`, ps1.ID, ps1.ID)
+	require.NoError(t, err)
```

Comment:
Could this SQL be encapsulated in a (test-only) ORM _method_?

Reply:
I could do it, though I'm not really a fan of adding methods to the ORM for testing purposes when we can just have a helper function. Any benefits on having this as an ORM method against just a helper test function? @jmank88

Reply:
It makes it easier to read and potentially test as well. What about a test-only method, declared in a `_test.go` file? It won't be in the public API, even if you export it.

Reply:
@vyzaldysanchez please implement this -- I think it'll be much clearer that way. The method will also be self-documenting by virtue of its name.

Reply:
I tried addressing this in [8e516d3](https://github.com/smartcontractkit/chainlink/pull/12367/commits/8e516d34395db5b2105c6482004974be5352db56). Please let me know of any improvements that can be made.